### ISSUE TITLE
SBBS address on main receive screen

### ISSUE THREAD

**Issue by Maxnflaxl** [BeamMW/android-wallet#643]
**Is your feature request related to a problem? Please describe.**
Show the SBBS on the main receive screen too, like in the desktop GUI wallet.

**Comment by freQniK** [comment]
yes. this would be a good improvement. I had to expand it yesterday to do a withdrawal from a CEX. 

The reason I'm commenting on all of these is I'm hoping to have some time soon to load the code in Android Studio.
